src/ape_cache/_cli.py

```diff
@@ -0,0 +1,82 @@
+import click
+import pandas as pd
+
+from ape import networks
+from ape.cli import NetworkBoundCommand, network_option
+from ape.logging import logger
+from ape.utils import ManagerAccessMixin
+
+
+def get_engine():
+    return ManagerAccessMixin.query_manager.engines["cache"]
+
+
+@click.group(short_help="Query from caching database")
+def cli():
+    """
+    Manage query caching database.
+    """
+
+
+@cli.command(short_help="Initialize a new cache database")
+@network_option()
+def init(network):
+    """
+    Initializes an SQLite database and creates a file to store data
+    from the provider.
+
+    Note that ape cannot store local data in this database. You have to
+    give an ecosystem name and a network name to initialize the database.
+    """
+
+    provider = networks.get_provider_from_choice(network)
+    ecosystem_name = provider.network.ecosystem.name
+    network_name = provider.network.name
+
+    get_engine().init_database(ecosystem_name, network_name)
+    logger.success(f"Caching database initialized for {ecosystem_name}:{network_name}.")
+
+
+@cli.command(
+    cls=NetworkBoundCommand,
+    short_help="Call and print SQL statement to the cache database",
+)
+@network_option()
```

Comment:
```suggestion
@network_option(required=True)
```

Reply:
This one isn't required because it should work like `ape console`